tests/src/main/java/com/jfrog/ide/eclipse/scan/GradleScanManagerTest.java

```diff
@@ -0,0 +1,99 @@
+package com.jfrog.ide.eclipse.scan;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.runtime.CoreException;
+
+import com.jfrog.ide.eclipse.utils.GradleArtifact;
+import com.jfrog.ide.eclipse.utils.Utils;
+
+import junit.framework.TestCase;
+
+public class GradleScanManagerTest extends TestCase {
+
+	public void testIsApplicable() throws CoreException, IOException {
+		IProject project = getGradleProject("gradleIsApplicable");
+		assertTrue(GradleScanManager.isApplicable(project));
+	}
+
+	public void testIsNotApplicable() throws CoreException, IOException {
+		IProject project = getGradleProject("gradleIsNotApplicable");
+		assertFalse(GradleScanManager.isApplicable(project));
+	}
+
+	public void testCreateGradleFile() throws IOException, CoreException {
+		IProject project = getGradleProject("gradleIsApplicable");
+		GradleScanManager gradleScanManager = new GradleScanManager(project);
+		String gradleFileLocation = Utils.getGradleScriptFileLocation(gradleScanManager);
+		File gradleScriptExpectedLocation = new File(
+				System.getProperty("user.home") + File.separator + "jfrog-eclipse-plugin" + File.separator
+						+ GradleScanManager.VERSION + File.separator + GradleScanManager.GRADLE_FILE_NAME);
```

Comment:
In all file, please consider to use Paths.get(...)

Reply:
Done also changed in the utils file as well.